projects/klippa/ngx-enhancy-forms/src/lib/elements/file-input/file-input.component.ts

```diff
@@ -16,6 +16,7 @@ export class FileInputComponent extends MultipleValueAccessorBase<File> {
 	@Input() onlyShowUploadButton = false;
 	@Input() useFullParentSize = false;
 	@Input() buttonText: string = 'Upload a file';
+	@Input() disabled: boolean = false;
```

Comment:
valueAccessorBase already has this property, which you are extending from

Reply:
indeed i only checked the parent for the inputs. Fixed in the next commit